metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlGetPartition.java

```diff
@@ -573,6 +574,10 @@ public int getPartitionCount(
                     null, null,
                     null, sort, pageable, forceDisableAudit);
             }
+        } catch (QueryTimeoutException e) {
+            // On query timeouts, propagate the error.
+            registry.counter(HiveMetrics.CounterHiveGetTablePartitionsTimeoutFailure.getMetricName()).increment();
+            throw e;
```

Comment:
Can we add the table tag into this metric? Not sure how often this happens, and it will help us trace the heavy loaded query/tables.